src/operator/nn/upsampling-inl.h

```diff
@@ -48,8 +48,8 @@ enum UpSamplingMultiInputMode {kConcat, kSum};
 }  // namespace up_enum
 
 struct UpSamplingParam : public dmlc::Parameter<UpSamplingParam> {
-  index_t scale;
-  index_t num_filter;
+  int scale;
+  int num_filter;
```

Comment:
`size_t`?

Reply:
I think size_t is usually not preferred in data-structure members, because it has a cross-platform issue

Reply:
@apeforest hm, yes, that might be an issue for interoperability. But for implementation use cases it wouldn't matter. `int` or maybe `unsigned int` (or even `uint23_t`) is fine as well.

Reply:
Yes, you are right. According to [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Integer_Types) `int` is actually preferred than `unsigned int` in this case.